R/plot_shinygosling.R

```diff
@@ -54,11 +54,13 @@ plot_granges <-
     )
     stopifnot(
```

Comment:
The convention in the package uses `isScalarCharacter()`, along the lines of (check that these are correct with respect to `NA` and `nzchar()`...)
```r
stopifnot(
    isScalarCharacter(title),
    isScalarCharacter(subtitle)
)
```

Reply:
57cbc97